package.json

```diff
@@ -11,16 +11,26 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-responsive-carousel": "^3.2.23",
```

Comment:
please remove the installed packages if unused 🧹

Reply:
We didn't have the time to make a proper carousel. Therefore, the fastest way was the horizontal scroll. We talked with the designer about it, and it was okay for her.
We might consider adding a carousel when we find the time to change it. 

Tailwind itself was fun. It made sense to us to use Tailwind. It's nice to add something new to our skill set.